### DISCUSSION TITLE
有线传输和wifi传输速度存在差异

### DISCUSSION THREAD

**Discussion by 9dog** [messense/aliyundrive-webdav#199] [Q&A] [answered]
### 问题描述

路由器AX3600
PC（有线）用RaiDrive测试下载速度能跑到10M+
ios（wifi）用infuse内的测速得到的速度在4-6M左右

（在AX3600上用过go-aliyundrive-webdav，ios和PC下载速度差不多，都是10M+。
（在Windows下使用docker执行webdav-aliyundriver的时候ios和PC的速度是一致的。

### 重现步骤

.

### 版本

1.0.0

### 运行平台

AX3600

### 日志

.

**Comment by 9dog** [comment]
（手机就放在路由器旁边测的。

**Comment by messense** [comment, marked as answer]
有线能够跑到 10M+ 说明瓶颈不在 aliyundrive-webdav 服务端，WiFI 慢可能的因素很多，建议先用普通的网络测速软件比如 https://speedtest.net 进行测试排除网络本身的速度问题。

**Comment by 9dog** [comment]
用过speedtest.net测速了，是500M宽带能跑到的正常速率。
我这边再找找自己的原因吧，反馈过来是想看看有没有人和我遇到一样的问题，想找找解决方法。
